core/server/master/src/main/java/alluxio/master/meta/MetaMasterClientServiceHandler.java

```diff
@@ -124,6 +125,17 @@ public void getMasterInfo(GetMasterInfoPOptions options,
                   Arrays.asList(ServerConfiguration.get(PropertyKey.ZOOKEEPER_ADDRESS).split(",")));
             }
             break;
+          case RAFT_ADDRESSES:
```

Comment:
You could replace this case with this code in case `PropertyKey.MASTER_EMBEDDED_JOURNAL_ADDRESSES` is not set.
```
if (ServerConfiguration.get(PropertyKey.MASTER_JOURNAL_TYPE)
                .equals(JournalType.EMBEDDED.name())) {
              List<String> raftAddresses =
                  ((RaftJournalSystem) mMetaMaster.getMasterContext().getJournalSystem())
                      .getQuorumServerInfoList().stream().map(info ->
                          String.format("%s:%d", info.getServerAddress().getHost(),
                              info.getServerAddress().getRpcPort())).collect(Collectors.toList());
              masterInfo.addAllRaftAddress(raftAddresses);
}
```

Reply:
done